Modules/AuctionManager/GUI.lua

```diff
@@ -319,6 +327,9 @@ function AuctionManagerGUI:Create()
     f:AddChild(CreateOptions(self))
     f:AddChild(CreateBidWindow(self))
 
+    -- Clear active bid on close
+    f:SetCallback('OnClose', function() AuctionManagerGUI:ClearSelectedBid(self) end)
```

Comment:
I don't think we should do this. If someone accidentaly closes the window the bids will clear during the auction.